src/main/java/com/google/devtools/build/lib/bazel/rules/android/AndroidNdkRepositoryFunction.java

```diff
@@ -131,6 +133,13 @@ private static String createBuildFile(
                 .replace("%name%", entry.getKey())
                 .replace("%fileGlobPattern%", entry.getValue()));
       }
+
+      // Create the Vulkan validation layers libraries
+      for (CToolchain toolchain : crosstool.getToolchainList()) {
+        vulkanValidationLayers.append(
+            vulkanValidationLayersTemplate.replace("%cpu%", toolchain.getTargetCpu()));
```

Comment:
This is not a unique name, if `toolchain.getTargetCpu()` is common between the outer crosstools loop.

Reply:
Thanks for this, I've changed the rule name to contain the toolchain name, and it seems to have fixed the tests.